containers/daap-python-base/src/var/task/data_platform_api_responses.py

```diff
@@ -18,6 +18,13 @@ def response_status_400(error) -> dict:
     return formatted_response
 
 
+def response_status_403(error) -> dict:
```

Comment:
Just a heads up, @tom-webber is looking at changing this module so you might need to rework it depending on what gets merged first https://github.com/ministryofjustice/data-platform/pull/1763

Reply:
I'm happy to wait and amend as saw the convo after i'd done this